### PR TITLE
02: Redundant options removed from Menu > Options

### PR DESCRIPTION
**Pull request by cjakeman**
See https://blueprints.launchpad.net/or/+spec/update-menu-options
and http://www.elvastower.com/forums/index.php?/topic/35301-menu-options/

### CODE REVIEW

File: Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs [hunk 1]
```diff
@@ -1116,62 +1113,51 @@ public virtual void UpdateTunnelForce()
...
             if (Train.IsPlayerDriven)   // Only calculate tunnel resistance when it is the player train.
             {
-                if (TunnelResistanceDependent)
+                if (CarTunnelData.FrontPositionBeyondStartOfTunnel.HasValue)
                 {
-                    if (CarTunnelData.FrontPositionBeyondStartOfTunnel.HasValue)
-                    {
+                    float? TunnelStart;
+                    float? TunnelAhead;
+                    float? TunnelBehind;
```
Comment (Sharpe49) [inline Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs:1120]: Those variables don't seem to be used anymore.

File: Source/Orts.Simulation/Common/Events.cs [hunk 2]
```diff
@@ -352,6 +340,9 @@ public static Event From(bool mstsBinEnabled, Source source, int eventID)
                         case 62: return Event.UncoupleB;
                         case 63: return Event.UncoupleC;
                         // Event 64 is unused in MSTS.
+                        // MSTSBin codes 66, 67 (documented at http://mstsbin.uktrainsim.com/)
+                        case 66: return Event.Pantograph2Up;
+                        case 67: return Event.Pantograph2Down;                        // Event 25 is possibly a vigilance reset in MSTS sound files but is never used.
```
Comment (twpol) [inline Source/Orts.Simulation/Common/Events.cs:345]: Looks like the event 25 comment from earlier has ended up on the end of this line by mistake

File: Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs (hunk 1 above)
Comment (cjakeman) [reply Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs:1120]: Thanks.
Fixed in next commit

File: Source/Orts.Simulation/Common/Events.cs (hunk 2 above)
Comment (cjakeman) [reply Source/Orts.Simulation/Common/Events.cs:345]: Thanks.
Fixed in next commit

File: Source/Documentation/Manual/cabs.rst [hunk 3]
```diff
@@ -16,8 +16,50 @@ OR supports with a high degree of compatibility all functions available in
...
 
-OR adds support for the ETCS circular speed gauge, as described 
-:ref:`here <options-etcs>`.
+
+ETCS circular speed gauge
+-------------------------
+
+When this option is selected, it is possible to add to the cabview a
+circular speed gauge accordingly to the European standard train control
+system ETCS.
```
Comment (pzgulyas) [inline Source/Documentation/Manual/cabs.rst:25]: This description got copied here by accident, I think. If I understand well, this option got removed from the code actually.

**Comment by cjakeman** [comment]
THis PR replaced by multiple PRs, one for each redundant option.

**Pull request closed without merging**